ansible/configs/sap-smart/post_software.yml

```diff
@@ -7,11 +7,12 @@
     - debug:
         msg: "Post-Software tasks Started"
 
-    - name: Dump Ansible Tower License to a file
-      copy:
-        content: long-lived-controller-manifest.zip
-        dest: /tmp/long-lived-controller-manifest.zip
-      when: tower_instance_count > 0
+    - name: downloading tower manifest from OpenTLC
+      get_url:
+        url: "{{ tower_manifest.url }}"
+        dest: "/tmp/{{ tower_manifest.url | regex_search('(\\w+)(\\.\\w+)+(?!.*(\\w+)(\\.\\w+)+)') }}"
```

Comment:
Should be able to reference var from `default_vars.yml`:
```
        dest: "{{ ansible_tower.install.manifest_file }}"
```

Reply:
thanks updated